saleor/product/utils.py

```diff
@@ -97,32 +99,61 @@ def products_for_cart(user):
     return products
 
 
-def get_variant_picker_data(variants, attributes):
-    data = {'attributes': [], 'variants': []}
-    for attribute in attributes:
-        data['attributes'].append({
+def get_variant_picker_data(product, discounts=None):
+    availability = get_availability(product, discounts)
+    variants = product.variants.all()
+    data = {'productAttributes': [], 'variantAttributes': [], 'variants': []}
```

Comment:
I lack frontend skills to answer your question ;) But seriously, I have the component that renders the price and variant attributes. How can I have the part with product attributes in between rendered by template?